R/utilities-units.R

```diff
@@ -287,6 +288,21 @@ getDimensionByName <- function(name) {
   rClr::clrCall(dimensionTask, "DimensionByName", enc2utf8(name))
 }
 
+#' Returns a named list with two entries (name, unit) corresponding to the name and unit
+#' extracted out of the `text` provided as parameter
+#'
+#' @examples
+#' res <- extractNameAndUnit("Value [mg]")
+#' # res$name == "Value", res$unit == "mg"
```

Comment:
@PavelBal This example does not help?
How would you name this? The goal of the implementation is to be able to split a string and return the variable part (name) and the unit part (unit)
Say your text is "Concentration [mg]". It returns "Concentration" and "mg" a